donjo-app/views/sid/wilayah/ajax_wilayah_maps.php

```diff
@@ -1,3 +1,18 @@
+<!--
+ * Modul Pemetaan di OpenSID
+ *
+ * /donjo-app/views/sid/wilayah/ajax_wilayah_maps.php v2.0 01/07/2020
+ *
+ * https://github.com/OpenSID/OpenSID
+ *
+ * Copyright (c) 2009 - 2015 Combine Resource Institution (http://lumbungkomunitas.net/)
+ * Copyright (c) 2016 - 2020 Perkumpulan Desa Digital Terbuka (https://opendesa.id)
+ *
+ * Dual licensed under the MIT and GPL licenses:
+ *   http://www.opensource.org/licenses/mit-license.php
+ *   http://www.gnu.org/licenses/gpl.html
+ -->
+
```

Comment:
Saya akan lihat contoh dari Codeigniter. Dan juga kita perlu cari cara untuk memasang info ini secara otomatis di setiap file. Mungkin ada solusi yg cocok di https://stackoverflow.com/questions/151677/tool-for-adding-license-headers-to-source-files. Supaya kalau ada perubahan, mudah untuk mengganti semua file sekaligus.